src/IB/IBFEMethod.cpp

```diff
@@ -1785,7 +1781,19 @@ void IBFEMethod::endDataRedistribution(Pointer<PatchHierarchy<NDIM> > /*hierarch
     // if we are not initialized then there is nothing to do
     if (d_is_initialized)
     {
-        // This is where we will repartition in a future commit
+        // At this point in the code SAMRAI has already redistributed the
```

Comment:
Should we add a runtime option to turn on/off dynamic repartitioning of libMesh data?

Reply:
Absolutely. I will move the checklist from #396 to here (which included such items).